x-pack/plugins/stack_connectors/public/connector_types/openai/dashboard_link.tsx

```diff
@@ -20,9 +20,9 @@ interface Props {
 export const DashboardLink: React.FC<Props> = ({
   connectorId,
   connectorName,
-  selectedProvider = '',
+  selectedProvider = 'OpenAI',
```

Comment:
I needed separate `DashboardLink` components for OpenAI and Bedrock in order to default the selectedProvider when rendered from the `actionReadOnlyExtraComponent` used in preconfigured connector edit form